integrated_channels/cornerstone/client.py

```diff
@@ -113,6 +117,16 @@ def create_course_completion(self, user_id, payload):
                 'Content-Type': 'application/json'
             }
         )
+        duration_seconds = time.time() - start_time
+        self.IntegratedChannelAPIRequestLogs.store_api_call(
```

Comment:
[Question] Any good reason to store this as an attribute within class?

Reply:
nope, I can move it outside the class - which I suggested to Mueez as well 😄

Reply:
I tried moving it outside the class but got `AppRegistryNotReady("Models aren't loaded yet.")`. Now placing `get_model` inside `create_course_completion`